core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/Tenant.java

```diff
@@ -32,8 +35,10 @@
 
     private String id;
 
+    @JsonInclude(Include.NON_EMPTY)
     private List<AggregationTemplate> aggregationTemplates = new ArrayList<>();
```

Comment:
Even if we only provide a single implementation, separating API from the impl is a cleaner approach IMO and makes it easier for people to consume and also makes it easier to test. Nobody wants core-api alone, but that's all client's should have to worry about. It also makes it possible to provide alternative implementations if someone were so inclined.